rcljava_common/src/main/java/org/ros2/rcljava/interfaces/ActionDefinition.java

```diff
@@ -0,0 +1,18 @@
+/* Copyright 2019 Open Source Robotics Foundation, Inc.
```

Comment:
Would it be ok with OSRF if the copyright notice is the same as the rest of the files?

Reply:
For this specific file it should no problem to use the same copyright notice since the actual code snippet is minimal / trivial. In other case where the contributions is significant I think the copyright notice should reflect the entity providing it.

Reply:
In rviz we would also have multiple copyright claims when there were large changes, e.g.:

https://github.com/ros2/rviz/blob/5595a69398df4d93aea711d408772ea34c53c230/rviz_common/include/rviz_common/factory/factory.hpp#L2-L4